### DISCUSSION TITLE
Is there a way to completely reset everything that has been setup falsely in vscode?

### DISCUSSION THREAD

**Discussion by F0rce** [coder/code-server#4208] [Q&A] [answered]
I think the title explains my question pretty simple. I have some major problems / bugs (probably caused by me) and I'm just curious if there is an easy way to completely wipe my installation without having to reinstall my whole server.

Thanks a lot,
David

**Comment by jsjoeio** [comment, marked as answer]
Hmm...I mean it's a bit manual, but you could:
- remove all settings -> Command Palette > Settings (JSON) -> delete everything
- uninstall all extensions
- remove -> Command Palette -> Keyboard Shortcuts (JSON) -> delete everything
- edit your `config.yaml` 

```yaml
bind-addr: 127.0.0.1:8080 
cert: false
```
